src/main/java/edu/harvard/iq/dataverse/util/SystemConfig.java

```diff
@@ -1056,4 +1056,8 @@ public boolean isDatafileValidationOnPublishEnabled() {
 	public boolean directUploadEnabled(Dataset dataset) {
     	return Boolean.getBoolean("dataverse.files." + dataset.getDataverseContext().getEffectiveStorageDriverId() + ".upload-redirect");
 	}
+	
+	public String getUserIpAddressSourceHeaderField() {
+        return System.getProperty("dataverse.useripaddresssourceheader", null);
```

Comment:
null is the default default so I think you can use System.getProperty("dataverse.useripaddresssourceheader");